tensorrt_llm/_torch/auto_deploy/utils/sharding_utils.py

```
@@ -54,43 +64,94 @@ def _load_hook_remove(
+
+    # get the subgraph of this module. Subgraph boundary is the next linear node.
+    next_lin_node, _ = bfs(node, is_any_lin_op, include_root=False)
+    nodes_to_validate = subgraph(
+        [node],
+        [next_lin_node],
+        include=lambda n: is_op(n, [torch.ops.aten.view, torch.ops.aten.reshape]),
+    )
+    for view_node in nodes_to_validate:
+        if len(view_node.args) < 2:
+            continue
+        if "sharded" in view_node.meta and view_node.meta["sharded"]:
```

Ruff (0.14.2)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)